js/base.js

```diff
@@ -0,0 +1,33 @@
+import { API as BaseAPI, mix, load, conf } from "yonius";
+import { OrderAPI } from "./order";
+
+const TEMPLATING_SERVICE_URL = "https://ripe-templating-now.platforme.com/";
+
+export class API extends mix(BaseAPI).with(OrderAPI) {
+    constructor(kwargs = {}) {
+        super(kwargs);
+        this.baseUrl = conf("TEMPLATING_SERVICE_URL", TEMPLATING_SERVICE_URL);

```

Comment:
let's call this config variable `PERI_TEMPLATING_URL` and the constant `BASE_URL`